iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/conf/CommonConfig.java

```diff
@@ -119,6 +119,9 @@ public class CommonConfig {
    */
   private long[] tierTTLInMs = {Long.MAX_VALUE};
 
+  /** The maximum number of TTL rules stored in the system, the default is 1000. */
+  private int TTLRuleCount = 1000;
```

Comment:
Still not precise enough. 
Usually, "count" means the current number you have.
In this case, it should probably be "capacity".

Reply:
Resolved. Change to `ttlRuleCapacity`.